index.md

```diff
@@ -1,8 +1,5 @@
 NOTE: This content is geared towards community engagement. Please refer to version specific product documentation for details.
 
-#### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)
```

Comment:
This link is still relevant and cannot be directly accessed from the product. Seems like it's still good to include.